client/server/middleware/test/logger.js

```diff
@@ -9,12 +9,14 @@ import EventEmitter from 'events';
 import loggerMiddleware from '../logger';
 import config from 'calypso/config';
 
-const mockLogger = {
+const requestLogger = {
 	info: jest.fn(),
-	child: jest.fn( () => mockLogger ),
+};
+const mockRootLogger = {
```

Comment:
Not strictly related to this change, but a fix suggested in https://github.com/Automattic/wp-calypso/pull/46838#discussion_r514264254